README.md

```diff
@@ -24,28 +64,38 @@ Grab via Maven:
   <version>3.2.1</version>
 </dependency>
 ```
-or Gradle:
+
+* _Gradle_
+
 ```groovy
 apt 'com.contentful.vault:compiler:3.2.1'
 compile 'com.contentful.vault:core:3.2.1'
 ```
-or Gradle 3.+:
+* _Gradle 3.+_
 ```groovy
 annotationProcessor 'com.contentful.vault:compiler:3.2.1'
 annotationProcessor 'com.contentful.vault:core:3.2.1'
 compile 'com.contentful.vault:core:3.2.1'
 ```
 
-Note for Gradle users, make sure to use the [android-apt][apt] Gradle plugin, which lets you configure compile-time only dependencies.
-Note for Gradle 3.0 users, please use the `annotationProcessor` instead of `apt`.
+> Note for Gradle: Use the [android-apt][apt] Gradle plugin, which lets you configure compile-time only dependencies.
```

Comment:
|before|after|
|-----|-----|
|which lets you configure compile-time only dependencies| which configures compile-time dependencies only.|